App.tsx

```diff
@@ -1,48 +1,31 @@
-import React, {useState} from 'react';
-import {SafeAreaView, View, FlatList, TouchableOpacity} from 'react-native';
-import Crypto from './src/components/Crypto';
-import {Container, Title, ImageApp, Space, BTNAddCrypto} from './styles';
-import {CryptoType} from './src/types/crypto';
-import {API_URL} from '@env';
+import AddCryptos from './src/Screens/AddCryptos/index';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import React from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import Home from './src/Screens/Home';
+import 'react-native-gesture-handler';
+import store from './src/store';
+import {Provider} from 'react-redux';
 
-const App = () => {
-  const [cryptos, setCryptos] = useState<CryptoType[]>([]);
+const Stack = createNativeStackNavigator();
 
-  const getCryptos = async () => {
-    const response = await fetch(
-      `${API_URL}?fields=id,name,slug,symbol,metrics/market_data`,
-    );
-    const data = await response.json();
-    setCryptos(data.data);
-  };
-
-  return (
-    <>
-      <SafeAreaView>
-        <Container>
-          <View>
-            <Title> CryptoTracker Pro </Title>
-          </View>
-          <ImageApp source={require('./src/assets/images/Face.jpg')} />
-        </Container>
-        <Space />
-        <View>
-          <View>
-            <FlatList
-              data={cryptos}
-              keyExtractor={({id}) => `${id}`}
-              renderItem={({item}) => <Crypto crypto={item} />}
-              ListFooterComponent={
-                <TouchableOpacity onPress={getCryptos}>
-                  <BTNAddCrypto> + Add a Cryptocurrency </BTNAddCrypto>
-                </TouchableOpacity>
-              }
-            />
-          </View>
-        </View>
-      </SafeAreaView>
-    </>
-  );
-};
+const App = () => (
+  <Provider store={store}>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen
+          name="Home"
+          component={Home}
+          options={{headerShown: false}}
```

Comment:
instead of using headerShown: false in all the screens, use it only in the stack navigator:

```
<Stack.Navigator screenOptions={{headerTitle: 'Test', headerShown: true}}  initialRouteName="Home">
```